overlord/boot/firstboot.go

```diff
@@ -148,11 +113,11 @@ func readAsserts(fn string, batch *assertstate.Batch) ([]*asserts.Ref, error) {
 }
 
 func importAssertionsFromSeed(st *state.State) error {
-	st.Lock()
-	defer st.Unlock()
-
 	assertSeedDir := filepath.Join(dirs.SnapSeedDir, "assertions")
 	dc, err := ioutil.ReadDir(assertSeedDir)
+	if os.IsNotExist(err) {
```

Comment:
notice that in that case there will be no model though, anyway I suppoe prepare-image atm always put that dir there